cmd/doc.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Commands and Sub-commands
+// =========================
+// Commands need to provide an `Info` method that returns an Info struct.
+
```

Comment:
This should start with `Package cmd...`

Also remove the weird markdown formatting.

Reply:
+1. Plus I'm guessing that means removing the markdown formatting throughout the remainder of the doc.